viewer/packages/360-images/src/Image360Facade.ts

```diff
@@ -60,7 +60,14 @@ export class Image360Facade<T> {
   }
 
   public preload(entity: Image360Entity, revision: Image360RevisionEntity, lockDownload?: boolean): Promise<void> {
-    return this._image360Cache.cachedPreload(entity, revision, lockDownload);
+    const annotationPromise = revision.getAnnotations();
+    const cacheLoadPromise = this._image360Cache.cachedPreload(entity, revision, lockDownload);
+
+    return awaitPromises();
+
+    async function awaitPromises(): Promise<void> {
+      await Promise.all([annotationPromise, cacheLoadPromise]);
```

Comment:
Do we care about the order of resolving these promises here?

Reply:
Not really, no... Do you mean I should be using some other function to await them?

Reply:
No, just curious since before this PR the order of things was defined explicitly as far as I understood.